google/cloud/bigtable/CMakeLists.txt

```diff
@@ -202,7 +202,8 @@ add_library(bigtable_client
             table_config.cc
             table_strong_types.h
             version.h
-            version.cc)
+            version.cc
+            poll_longrunning_operation.h)
```

Comment:
please move poll_longrunning_operation.h to keep the list of files alphabetical.